javagen/src/main/java/com/azure/autorest/template/PomTemplate.java

```diff
@@ -31,6 +31,12 @@ public static PomTemplate getInstance() {
     }
 
     public final void write(Pom pom, XmlFile xmlFile) {
+        // copyright
+        xmlFile.blockComment(xmlLineComment -> {
+            xmlLineComment.line(" ~ Copyright (c) Microsoft Corporation. All rights reserved.");
+            xmlLineComment.line(" ~ Licensed under the MIT License.");
```

Comment:
We probably should use content here https://github.com/Azure/autorest.java/blob/main/extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java#L545-L547

Reply:
changed

Reply:
LGTM.

@srnagar  Does it look good to you?